lib/system/api/network.toit

```diff
@@ -110,65 +111,8 @@ interface NetworkService:
   socket-mtu handle/int -> int
   static SOCKET-MTU-INDEX /int ::= 306
 
-class NetworkServiceClient extends ServiceClient implements NetworkService:
+class NetworkServiceClient extends NetworkServiceClientBase:
```

Comment:
This is only used from lib/net/net.toit and a single test now. All other uses have been changed to `NetworkServiceClientBase`.